src/main/resources/view/HelpWindow.css

```diff
@@ -1,4 +1,4 @@
-#copyButton, #helpMessage {
+#copyButton, #urlMessage, #addCommand, #clearCommand, #deleteCommand, #editCommand, #exitCommand {
```

Comment:
As discussed earlier, could experiment with using `styleClass` and css classes to achieve DRY :)

Reply:
I've grouped the relevant elements together into styleClasses like headers and container, thanks for the tip!